### PR TITLE
Make default_volume_type configurable

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb [hunk 1]
```diff
@@ -337,6 +337,9 @@ default_availability_zone = <%= @default_availability_zone %>
 
 # Default volume type to use (string value)
 #default_volume_type = <None>
+<% unless @default_volume_type.nil? or @default_volume_type == "" -%>
```
Comment (aplanas) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: I _think_ that both checks can be put together with `@default_volume_type.to_s.empty?` or with the equivalent `@default_volume_type.to_s == ""?`

**Comment by aplanas** [comment]
+1

**Comment by toabctl** [comment]
-1 Commit message wrong. It's ```default_volume_type```, not availability

**Comment by dirkmueller** [comment]
@toabctl fixed, thanks

**Comment by toabctl** [comment]
+1

**Pull request merged**